User-Data Store Migration/dump-converter.py

```diff
@@ -264,8 +277,6 @@ def finalize(out):
                                 break
                             else:
                                 line = newLine
-                else:
-                    writeline(dst, line)

```

Comment:
this was the default write to the output of everything not processed

Reply:
as far as I understand, this is the bugfix

Reply:
this is the main change to prevent keeping the old lines, the other changes are needed as well to ensure that I explicitly write what before I was keeping from the input